aunt-leahs-app/src/pages/adminLocations.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Head from '../components/header.jsx';
+import constants from '../constants';
+import '../styles.css';
+import { LocationList } from '../components/adminLocationList/locationList';
+import TabComponent from '../components/tab';
+
+const AdminLocationsPage = () => (
+    <div className="homepage">
+        <div style={{display:'flex', flexDirection:'row', alignItems: 'center'}}>
+            <Head />
+            <TabComponent tabs={[
+                { title: 'Shift Data', content: ''},
+                { title: 'Volunteer Data',content: ''},
+                { title: 'Locations', content: ''},
+                { title: 'Sign Out', content: ''}
+            ]} />
+        </div>
+        
+        <div className="homepage-list">
```

Comment:
This class doesn't seem to exist

Reply:
I just copied that from the homepage one (at the time). I'll remove it and see how it looks without